lsix

```diff
@@ -191,7 +191,7 @@ main() {
 	readarray -t < <(printf "%s\n" "$@" | sort)
 
 	# Only show first frame of animated GIFs if filename not specified.
-	for x in ${!MAPFILE[@]}; do
+	for x in "${!MAPFILE[@]}"; do
```

Comment:
Shell check is overly persnickety. It is impossible for the keys of an indexed array (which are just numbers) to have spaces.

Reply:
I will fix it by letting shellcheck ignore this.